erpnext/accounts/doctype/discounted_invoice/discounted_invoice.json

```diff
@@ -38,13 +38,11 @@
    "read_only": 1
   },
   {
-   "fetch_from": "sales_invoice.grand_total",
```

Comment:
No need to remove this. Just replace grand_total with outstanding_amount and check fetch if empty so that the user entered amount is not overridden . No need to write any client side code
![image](https://user-images.githubusercontent.com/42651287/65620716-f30e1c00-dfdf-11e9-94b3-a1178471ff22.png)